ociclient/client_test.go

```diff
@@ -36,29 +32,32 @@ var _ = Describe("client", func() {
 			defer ctx.Done()
 
 			ref := testenv.Addr + "/test/artifact:v0.0.1"
-			manifest := uploadTestManifest(ctx, ref)
+			manifest, _, err := testutils.UploadTestManifest(ctx, client, ref)
+			Expect(err).ToNot(HaveOccurred())
 
 			res, err := client.GetManifest(ctx, ref)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(res.Config).To(Equal(manifest.Config))
 			Expect(res.Layers).To(Equal(manifest.Layers))
 
-			compareManifestToTestManifest(ctx, ref, res)
+			// TODO: oci image index test only working because cache is filled in this function with config/layer blobs. should be fixed
```

Comment:
TODO resolved?

Reply:
unfortunately not :( 
I would do this in a different PR, as it also touches another part of coding which is not part of this PR